src/utils/utils.js

```diff
@@ -18,4 +18,20 @@ function difference(object, base) {
   return newValue
 }
 
-export { setIfValue, difference }
+function isJson(item) {
```

Comment:
Suggest the following. if `item` is an object, then it it is "json". if it is a string and parses then it too is "json". 

```javascript
function isJson(item) {
  if (!item) return false
  if (typeof item !== 'string') return true
  try {
    return !!JSON.parse(item)
  } catch (e) {
    return false
  }
}
```